### ISSUE TITLE
Fork and add maintainers?

### ISSUE THREAD

**Issue by edg-l** [lalrpop/lalrpop#712]
I wonder how willing other people are to create a new group and maintain this crate, I would do it but if I'm honest, I don't know much about the inner workings of this crare or about LR/LALR itself. I think it's a good one, if you know good alternatives I would like to know them too.

This could have been probably be managed better by increasing the bus factor: https://users.rust-lang.org/t/bus-factor-1-for-crates/17046

**Comment by youknowone** [comment]
I recently sent a email to maintainers and got a reply a few hours ago. If someone would like to help maintenance, please leave comment to become a volunteer. Based on the email, I guess those volunteers will be welcomed.

**Comment by discord9** [comment]
> I recently sent a email to maintainers and got a reply a few hours ago. If someone would like to help maintenance, please leave comment to become a volunteer. Based on the reply, I guess those volunteers will be welcomed.

Willing to help+1

**Comment by yannham** [comment]
Hi, this project has unfortunately been under-maintained for a few years. However, the original owner @nikomatsakis has started to dedicate a bit of time to LALRPOP again, and potentially  @youknowone as well.

We'll try to release 0.19.9 very soon (with urgent fixes, such as removing the warning on current stable Rust saying that LALRPOP will become incompatible with future version of Rust), and then walk toward a 1.0 on a slightly longer term.

**Comment by nikomatsakis** [comment]
I'm going to go ahead and close this issue, given that we've got some forward momentum again:

https://github.com/lalrpop/lalrpop/pull/717

But I want to encourage @discord9 and @edg-l to reach out to me (`niko@alum.mit.edu`). If you have some time, any help would be much appreciated. You don't need detailed knowledge of the LR algorithm for most of the work.

My initial goal was just to get LALRPOP to a 1.0 state with some minor fixes (and I still think that's a good next step), but if we got a few more folks, I'd love to discuss how we can make bigger usability improvements.
